Upstream-Status: Inappropriate [OE-Specific]

Provide compatibility with glibc versions > 2.27

Signed-off-by: Alejandro Enedino Hernandez Samaniego <aehs29@gmail.com>




Index: git/main.c
===================================================================
--- git.orig/main.c
+++ git/main.c
@@ -15,6 +15,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <sys/sysmacros.h>
 
 #include "rootdev.h"
 
Index: git/rootdev.c
===================================================================
--- git.orig/rootdev.c
+++ git/rootdev.c
@@ -24,6 +24,8 @@
 #include <unistd.h>
 #include <linux/btrfs.h>
 #include <linux/magic.h>
+#include <sys/sysmacros.h>
+
 
 /*
  * Limit prevents endless looping to find slave.
